index.js

```diff
@@ -3,6 +3,7 @@
  */
 
 import {AppRegistry} from 'react-native';
+import 'react-native-get-random-values';
```

Comment:
Why we need this as well? If need random values we can switch over to React 18 and can use useId hook.

Reply:
'react-native-get-random-values' is a peer dependency for uuid. useId in react 18 shouldn't be used for keys in a list as per the docs.

Note:
useId is not for generating [keys in a list](https://reactjs.org/docs/lists-and-keys.html#keys). Keys should be generated from your data.

[https://reactjs.org/blog/2022/03/29/react-v18.html#useid](https://reactjs.org/blog/2022/03/29/react-v18.html#useid)